### PR TITLE
Update OWNERS

### PR DESCRIPTION
**Pull request by Shreyanand**
SSIA

### CODE REVIEW

**Comment by codificat** [comment]
Do we have prow here?

In any case,
/lgtm

File: OWNERS [hunk 1]
```diff
@@ -1,20 +1,17 @@
...
+  - Shreyanand
+  - suppathak
+  - erikerlandson
+  - codificat
 
 reviewers:
   - aakankshaduggal
-  - chauhankaranraj
   - hemajv
-  - isabelizimm
```
Comment (suppathak) [inline OWNERS:13]: Hi @Shreyanand . Not sure if Isabel , Margaret, and karan are still active in the organization repo?

Comment (Shreyanand) [reply OWNERS:13]: I don't think they are and so removed them.

**Comment by Shreyanand** [comment]
> Do we have prow here?
> 
> In any case, /lgtm

We do have the [config](https://github.com/redhat-et/foundation-models-for-documentation/blob/master/.prow.yaml). Should we add Sesheta as well?

**Review by suppathak — COMMENTED** [review]
/lgtm 
Thanks @Shreyanand !!

**Comment by codificat** [comment]
> We do have the [config](https://github.com/redhat-et/foundation-models-for-documentation/blob/master/.prow.yaml). Should we add Sesheta as well?

TBH I don't know. I was just pointing out that the change in the PR seems to be only relevant to prow.

For review/approval we should be fine among us for the time being I think.

I believe it would be nice to have some tests and checks for the PRs, but I don't know what the preference is to implement them (prow or something else).

**Review by codificat — APPROVED** [review]
/lgtm
thanks!

**Pull request merged**